Makefile

```diff
@@ -58,8 +58,9 @@ endif
 	cd $(OLENA_DIR) && autoreconf -i
 
 deps-ubuntu:
-	apt-get -y install libmagick++-dev libgraphicsmagick++1-dev libboost-dev \
-		xmlstarlet ca-certificates
+	apt-get -y install \
+		g++ make libtool autoconf \
```

Comment:
I already have a list of packages required at build time [in Dockerfile](https://github.com/OCR-D/ocrd_olena/blob/3262fee8c180724fec56b1c69c3d686b97d47084/Dockerfile#L19): `g++ make automake git`.

You need `git` for (the default) `OLENA_VERSION=git`.

And `automake` implies `autoconf`, which implies `libtool`.

Reply:
I see, thanks. We're following/adapting the setup guide which already installed git, so I did not (but should) list it here as well. libtool < autoconf < automake is good to know too.